README.md

```diff
@@ -27,21 +27,19 @@ getLogs()
 | 547               | 34            | SalesProcess.csv                 | 2018-09-13T13:27:50.414Z   | Europe/Berlin  |
 
 ## Filter and performance / followers statistics
-```
-filter(logName)
-```
-#### Example
-```
-discoveredModel("Incident_withImpactAttributes.csv")
+#### Example Usage
+```R
+# With empty trace filter sequence
+discoveredModel(lanaURL, lanaToken, logId)
+
+# With non-empty trace filter sequence
+discoveredModel(lanaURL, lanaToken, logId, traceFilterSequence = "trace sequence")
 ```
 ## Activity Performance Statistics
-gives statistics about activities such as minimum/maximum duration, average/median/total duration, standard deviation and frequency.
+Gives statistics about activities such as minimum/maximum duration, average/median/total duration, standard deviation and frequency.
+#### Example Usage
 ```
-activityPerformance(logName)
-```
-#### Example
-```
-activityPerformance("Incident_withImpactAttributes.csv")
+activityPerformance(lanaURL, lanaToken, logId)
 ```
 
 | .id                           | frequency       | totalDuration   | minDuration     | maxDuration   | avgDuration    | standardDeviation| median         |
```

Comment:
activityPerfomance is not really used anymore, I would just delete it from the readme